src/core/Akka.TestKit/TestKitBase.cs

```diff
@@ -189,6 +188,30 @@ protected void InitializeTest(ActorSystem system, ActorSystemSetup config, strin
             _testState.TestActor = testActor;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static void WaitUntilTestActorIsReady(IActorRef testActor)
```

Comment:
Great catch - should we move this to its own PR so we can get a clean signal on this test library?

Reply:
Will do